sourcegraph/jetbrains
# Properly handle VSCode Positions

Fixes https://github.com/sourcegraph/jetbrains/issues/1623

## Test plan

1. Open sourcegraph/cody
2. Open agent/src/index.test.ts
3. Put cursor in some nested code. I used line with accessToken: 'REDACTED_0ba08837494d00e3943c46999589eb29a210ba8063f084fff511c8e4d1503909',
4. Try to run any edit command

It should fail before my fix and pass after.

Also, I added unittests :)
